buildSrc/src/minimumRuntime/java/org/elasticsearch/gradle/Version.java

```diff
@@ -51,6 +51,10 @@ public static Version fromString(final String s) {
         );
     }
 
+    public static Version current() {
```

Comment:
Maybe we should be more specific here? The `Version` class is generic, and not only used to model ES versions. We should either rename this to `currentElasticsearch()` or similar or move this to `VersionProperties` itself.

Reply:
I moved the method as suggested